### PR TITLE
Change name from LocalCommandlineCodeExecutor to LocalCommandLineCodeExecutor

### PR DESCRIPTION
**Pull request by jackgerrits**
## Why are these changes needed?

Fixes a small naming issue

## Checks

- [ ] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

### CODE REVIEW

File: autogen/coding/local_commandline_code_executor.py [hunk 1]
```diff
@@ -13,7 +13,8 @@
 
 
 __all__ = (
-    "LocalCommandlineCodeExecutor",
+    "LocalCommandLineCodeExecutor",
+    "LocalCommandLineCodeExecutor",
     "CommandlineCodeResult",
```
Comment (ekzhu) [inline autogen/coding/local_commandline_code_executor.py:18]: Also "CommandLineCodeResult" ?

File: autogen/coding/local_commandline_code_executor.py [hunk 2]
```diff
@@ -13,7 +13,8 @@
 
 
 __all__ = (
-    "LocalCommandlineCodeExecutor",
+    "LocalCommandLineCodeExecutor",
```
Comment (ekzhu) [inline autogen/coding/local_commandline_code_executor.py:16]: One of it should be the old class name?

File: autogen/coding/local_commandline_code_executor.py (hunk 1 above)
Comment (jackgerrits) [reply autogen/coding/local_commandline_code_executor.py]: Just "exporting" the new names

File: autogen/coding/local_commandline_code_executor.py (hunk 2 above)
Comment (jackgerrits) [reply autogen/coding/local_commandline_code_executor.py:16]: I think it makes sense to just export the new names

File: autogen/coding/local_commandline_code_executor.py (hunk 1 above)
Comment (jackgerrits) [reply autogen/coding/local_commandline_code_executor.py]: Good catch!

**Review by ekzhu — APPROVED** [review]

**Comment by ekzhu** [comment]
Just some context: we haven't documented this class anywhere yet. So it is a good time to make this type of changes. 

@sonichi @qingyun-wu for awareness and final approval.

**Comment by sonichi** [comment]
This change requires an openai test. Could you make the PR from the upstream repo?

**Comment by jackgerrits** [comment]
Moved to ##1859

**Pull request closed without merging**